src/components/views/dialogs/MessageEditHistoryDialog.js

```diff
@@ -82,7 +90,23 @@ export default class MessageEditHistoryDialog extends React.PureComponent {
     render() {
         let content;
         if (this.state.error) {
-            content = this.state.error;
+            const {error} = this.state;
+            if (error.errcode === "M_UNRECOGNIZED") {
+                content = (<p className="mx_MessageEditHistoryDialog_error">
+                    {_t("Your homeserver doesn't seem to support this feature.")}
+                </p>);
+            } else if (error.errcode) {
+                // some kind of error from the homeserver
+                content = (<p className="mx_MessageEditHistoryDialog_error">
+                    {_t("Something went wrong!")}
+                </p>);
```

Comment:
Should we log the error to the console and / or the dialog?

Reply:
did it in the loadMoreEdits method to not get repeated logs when rerendering.